ts/api/wallet/wallet-api.ts

```diff
@@ -192,4 +201,8 @@ export class WalletAPI {
   > {
     return transactions;
   }
+
+  public static getTransaction(id: number): Readonly<WalletTransaction> {
```

Comment:
Any reason why this is needed? the wallet is migrating towards a redux-based state with the information, so these mock API should exist no longer